trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -60,7 +77,7 @@ const addPartyUrns = async (deal, auditDetails) => {
       ...deal.tfm,
       parties: {
         exporter: {
-          partyUrn: await getPartyUrn({ companyRegNo: deal.exporter.companiesHouseRegistrationNumber }),
+          partyUrn: await getPartyUrn({ companyRegNo: deal.exporter.companiesHouseRegistrationNumber, companyName: deal.exporter.companyName }),
```

Comment:
I've not flagged this as that seems unnecessarily fiddly, let me know if I should!

Reply:
What do you mean by flagged please?